testsuite/config/openshift_loader.py

```diff
@@ -26,20 +26,18 @@ def load(obj, env=None, silent=True, key=None, filename=None):
         tools = client.change_project(obj["tools"]["project"])
     obj["tools"] = tools
 
-    service_protection = obj.setdefault("service_protection", {})
-    inject_client(service_protection, client, "project")
-    inject_client(service_protection, client, "project2")
+    # service_protection = obj.setdefault("service_protection", {})
```

Comment:
do something with this commented code. either delete or surround by if condition if needed in only some cases